### ISSUE TITLE
The +/- characters of zoom controls should be explicitly hidden from ATs

### ISSUE THREAD

**Issue by Malvoz** [Leaflet/Leaflet#7433]
## Motivation

When using a screen reader such as [NVDA](https://github.com/nvaccess/nvda), <q>Plus</q> is announced when hovering the Zoom In control, and <q>Minus</q> is announced when hovering the Zoom Out control:

<table>
  <tr>
    <th>Screen capture with sound</th>
  </tr>
  <tr>
    <td width="450"><video src="https://user-images.githubusercontent.com/26493779/106029117-0c8c1000-60cd-11eb-9041-e838de218f8f.mp4"></video>
    </td>
  </tr>
</table>

## Proposed solution

Both the `+` and `-` characters should be wrapped in `<span aria-hidden="true">` to prevent screen readers from announcing <q>Plus</q> and <q>Minus</q>:

https://github.com/Leaflet/Leaflet/blob/b52961aa469dc17b3710ef7a9c0f13b6f4dba550/src/control/Control.Zoom.js#L22-L23

https://github.com/Leaflet/Leaflet/blob/b52961aa469dc17b3710ef7a9c0f13b6f4dba550/src/control/Control.Zoom.js#L30-L31
